scripts/common/deploy-system.ts

```diff
@@ -15,7 +16,7 @@ import {
 } from '../../typechain'
 import { AddressRegistry } from './addresses'
 import { HardhatUtils } from './hardhat.utils'
-import { AutomationServiceName, Network, TriggerType, TriggerGroupType } from './types'
+import { AutomationServiceName, Network, TriggerGroupType } from './types'
```

Comment:
there is 
`export enum TriggerGroupType {
  ConstantMultiple = 1,
}`

in automtion package. I didn't even know :)